src/models/event.ts

```diff
@@ -136,15 +136,20 @@ export default class Event extends RestfulModel {
     }
   }
 
-  deleteRequestQueryString(params: { [key: string]: any } = {}) {
+  deleteRequestQueryString(
+    params: { [key: string]: any } = {}
```

Comment:
@mrashed-dev for such instances like `{ [key: string]: any }` - I think it might be better to opt to use the [Record<keyType, valueType>](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) built-in type.

For example `params: Recod<string,any>`.

Reply:
Separately, is there no way to avoid the use of `any`? It seems to defeat the purpose of typing :'(

If we do have to use `any`, could maybe [unknown](https://mariusschulz.com/blog/the-unknown-type-in-typescript) be of better use? (see article for more info)

Reply:
@AaronDDM thanks for bringing unknown to my attention! Let me play with it and see how it works with the different methods we have.

Reply:
@AaronDDM I've updated the PR to include both changes!